test/suites/migration.sh

```diff
@@ -424,6 +424,16 @@ migration() {
   # Test stateless copies
   lxc_remote copy --stateless l2:migratee/snap0 l1:migratee-new-name
 
+  if [ "${lxd_backend}" = "btrfs" ] && [ "${lxd2_backend}" = "btrfs" ] || [ "${lxd_backend}" = "zfs" ] && [ "${lxd2_backend}" = "zfs" ]; then
+    lxc_remote init testimage l1:c1
+    lxc_remote copy l1:c1 l2:c1
```

Comment:
Suggest also starting the instance after copied to check it starts, and adding a file inside it to check that got copied to prove the copy process is working.